src/panels/config/logs/ha-config-logs.ts

```diff
@@ -55,7 +55,6 @@ export class HaConfigLogs extends LitElement {
       : html`
           <div class="search">
             <search-input
-              autofocus
               no-label-float
               no-underline
```

Comment:
These two attributes should be removed too btw, `search-input` doesn't have them anymore

Reply:
https://github.com/home-assistant/frontend/pull/11889